src/test/frontend/replicasetlist/logsmenu_controller_test.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import LogsMenuController from 'replicasetlist/logsmenu_controller';
+import LogsMenuPodInfo from 'replicasetlist/logsmenupodinfo';
+
+describe('Logs menu controller', () => {
+  const inputReplicaSetPods = [
+    {
+      name: 'pod1',
+      startTime: '2015-12-08T08:59:11Z',
+      podContainers: [{name: 'container1', restartCount: 1}, {name: 'container2', restartCount: 8}],
+    },
+    {
+      name: 'pod2',
+      startTime: '2015-12-08T08:59:12Z',
+      podContainers: [{name: 'container3', restartCount: 1}, {name: 'container4', restartCount: 2}],
+    },
+    {
+      name: 'pod3',
+      startTime: '2015-12-08T08:59:13Z',
+      podContainers: [{name: 'container5', restartCount: 11}],
+    },
+  ];
+  const expectedResult = [
+    new LogsMenuPodInfo('pod3', '2015-12-08T08:59:13Z', 11),
+    new LogsMenuPodInfo('pod1', '2015-12-08T08:59:11Z', 9),
+    new LogsMenuPodInfo('pod2', '2015-12-08T08:59:12Z', 3),
+  ];
+  let ctrl;
```

Comment:
Currently I've moved all logic to backend. Controller just passes raw data from backend to view, so I removed this test because there is no logic here